index.md

```
@@ -0,0 +1,140 @@
+
+## Resources
+
+- **[resources/icon.png](./resources/icon.png)** - Icon image
+- **[resources/icon.ico](./resources/icon.ico)** - Windows icon
+- **[resources/omniview.png](./resources/omniview.png)** - OmniView logo
+
+## Configuration
+
+- **[settings.json](./settings.json)** - VS Code settings
+- **[skills-lock.json](./skills-lock.json)** - Skills lock file
+- ** [.gitignore](./.gitignore)** - Git ignore patterns
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)